### DISCUSSION TITLE
How to use the program to create a new agv car

### DISCUSSION THREAD

**Discussion by CSXYA** [openTCS/opentcs#73] [Q&A] [answered]
How to create a new agv' car by calling opentcs on the web while the program is running? And connect to

**Comment by swltr** [comment, marked as answer]
Please stick to English for posts in this discussion forum.

**Reply by CSXYA** [reply]
When the program is running, how to create a new agv through the program internal method

**Reply by swltr** [reply]
Vehicles are part of the plant model. The plant model is uploaded to the kernel before operation. The kernel currently does not support adding vehicles dynamically during operation. You would have to upload a new plant model for that (which results in transport order processing to be canceled).

**Reply by weibin06** [reply]
I figured if I could restore past orders when restarting the kernel, then adding vehicles by updating the map model wouldn't be a problem. Again, this is a problem if the previous transport order is restored when the kernel is restarted

**Reply by CSXYA** [reply]
I see. Thanks for the answer
